{"uid":"e95299886c917a92","name":"test_list_address_properties_country_10_326","fullName":"tests.test_checklist.TestPromotionChecklist#test_list_address_properties_country_10_326","historyId":"26a53e8216453ac74a13ce2cbc0049c0","time":{"start":1739698127726,"stop":1739698421937,"duration":294211},"status":"passed","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_unittest_setUpClass_fixture_TestPromotionChecklist","time":{"start":1739690686858,"stop":1739690686859,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"status":"passed","steps":[{"name":"Click authorisation button","time":{"start":1739698128496,"stop":1739698129165,"duration":669},"status":"passed","steps":[{"name":"Get authorisation button","time":{"start":1739698128496,"stop":1739698128507,"duration":11},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":1,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Source select","time":{"start":1739698129700,"stop":1739698129817,"duration":117},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"source","value":"'326'"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739698131153,"stop":1739698131246,"duration":93},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"1"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739698131247,"stop":1739698138881,"duration":7634},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"\nMetadata = PROPERTY Ok âœ…\n","time":{"start":1739698138914,"stop":1739698138914,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739698138935,"stop":1739698139064,"duration":129},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"2"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739698139065,"stop":1739698159131,"duration":20066},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x5575f861309a <unknown>\n#1 0x5575f80da8b0 <unknown>\n#2 0x5575f812bb99 <unknown>\n#3 0x5575f812bd21 <unknown>\n#4 0x5575f817a854 <unknown>\n#5 0x5575f81519ad <unknown>\n#6 0x5575f8177b8b <unknown>\n#7 0x5575f8151753 <unknown>\n#8 0x5575f811d38e <unknown>\n#9 0x5575f811eb51 <unknown>\n#10 0x5575f85dc76b <unknown>\n#11 0x5575f85e06f2 <unknown>\n#12 0x5575f85c88fc <unknown>\n#13 0x5575f85e12e4 <unknown>\n#14 0x5575f85ac9cf <unknown>\n#15 0x5575f8601cd8 <unknown>\n#16 0x5575f8601eb6 <unknown>\n#17 0x5575f8611f16 <unknown>\n#18 0x7fa15d09caa4 <unknown>\n#19 0x7fa15d129c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class CUSTOMPROPERTY is not mapped","time":{"start":1739698159132,"stop":1739698159132,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739698159154,"stop":1739698159273,"duration":119},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"3"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739698159273,"stop":1739698179331,"duration":20058},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x5575f861309a <unknown>\n#1 0x5575f80da8b0 <unknown>\n#2 0x5575f812bb99 <unknown>\n#3 0x5575f812bd21 <unknown>\n#4 0x5575f817a854 <unknown>\n#5 0x5575f81519ad <unknown>\n#6 0x5575f8177b8b <unknown>\n#7 0x5575f8151753 <unknown>\n#8 0x5575f811d38e <unknown>\n#9 0x5575f811eb51 <unknown>\n#10 0x5575f85dc76b <unknown>\n#11 0x5575f85e06f2 <unknown>\n#12 0x5575f85c88fc <unknown>\n#13 0x5575f85e12e4 <unknown>\n#14 0x5575f85ac9cf <unknown>\n#15 0x5575f8601cd8 <unknown>\n#16 0x5575f8601eb6 <unknown>\n#17 0x5575f8611f16 <unknown>\n#18 0x7fa15d09caa4 <unknown>\n#19 0x7fa15d129c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class DATASYSTEM is not mapped","time":{"start":1739698179332,"stop":1739698179332,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739698179354,"stop":1739698179468,"duration":114},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"4"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739698179468,"stop":1739698199506,"duration":20038},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x5575f861309a <unknown>\n#1 0x5575f80da8b0 <unknown>\n#2 0x5575f812bb99 <unknown>\n#3 0x5575f812bd21 <unknown>\n#4 0x5575f817a854 <unknown>\n#5 0x5575f81519ad <unknown>\n#6 0x5575f8177b8b <unknown>\n#7 0x5575f8151753 <unknown>\n#8 0x5575f811d38e <unknown>\n#9 0x5575f811eb51 <unknown>\n#10 0x5575f85dc76b <unknown>\n#11 0x5575f85e06f2 <unknown>\n#12 0x5575f85c88fc <unknown>\n#13 0x5575f85e12e4 <unknown>\n#14 0x5575f85ac9cf <unknown>\n#15 0x5575f8601cd8 <unknown>\n#16 0x5575f8601eb6 <unknown>\n#17 0x5575f8611f16 <unknown>\n#18 0x7fa15d09caa4 <unknown>\n#19 0x7fa15d129c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class ENUMERATION is not mapped","time":{"start":1739698199507,"stop":1739698199507,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739698199528,"stop":1739698199653,"duration":125},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"5"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739698199653,"stop":1739698219709,"duration":20056},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x5575f861309a <unknown>\n#1 0x5575f80da8b0 <unknown>\n#2 0x5575f812bb99 <unknown>\n#3 0x5575f812bd21 <unknown>\n#4 0x5575f817a854 <unknown>\n#5 0x5575f81519ad <unknown>\n#6 0x5575f8177b8b <unknown>\n#7 0x5575f8151753 <unknown>\n#8 0x5575f811d38e <unknown>\n#9 0x5575f811eb51 <unknown>\n#10 0x5575f85dc76b <unknown>\n#11 0x5575f85e06f2 <unknown>\n#12 0x5575f85c88fc <unknown>\n#13 0x5575f85e12e4 <unknown>\n#14 0x5575f85ac9cf <unknown>\n#15 0x5575f8601cd8 <unknown>\n#16 0x5575f8601eb6 <unknown>\n#17 0x5575f8611f16 <unknown>\n#18 0x7fa15d09caa4 <unknown>\n#19 0x7fa15d129c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class PROPERTYROOMS is not mapped","time":{"start":1739698219710,"stop":1739698219710,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739698219732,"stop":1739698219872,"duration":140},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"6"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739698219872,"stop":1739698239927,"duration":20055},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x5575f861309a <unknown>\n#1 0x5575f80da8b0 <unknown>\n#2 0x5575f812bb99 <unknown>\n#3 0x5575f812bd21 <unknown>\n#4 0x5575f817a854 <unknown>\n#5 0x5575f81519ad <unknown>\n#6 0x5575f8177b8b <unknown>\n#7 0x5575f8151753 <unknown>\n#8 0x5575f811d38e <unknown>\n#9 0x5575f811eb51 <unknown>\n#10 0x5575f85dc76b <unknown>\n#11 0x5575f85e06f2 <unknown>\n#12 0x5575f85c88fc <unknown>\n#13 0x5575f85e12e4 <unknown>\n#14 0x5575f85ac9cf <unknown>\n#15 0x5575f8601cd8 <unknown>\n#16 0x5575f8601eb6 <unknown>\n#17 0x5575f8611f16 <unknown>\n#18 0x7fa15d09caa4 <unknown>\n#19 0x7fa15d129c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class PROPERTYUNITTYPES is not mapped","time":{"start":1739698239928,"stop":1739698239928,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739698239957,"stop":1739698240083,"duration":126},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"7"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739698240083,"stop":1739698260127,"duration":20044},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x5575f861309a <unknown>\n#1 0x5575f80da8b0 <unknown>\n#2 0x5575f812bb99 <unknown>\n#3 0x5575f812bd21 <unknown>\n#4 0x5575f817a854 <unknown>\n#5 0x5575f81519ad <unknown>\n#6 0x5575f8177b8b <unknown>\n#7 0x5575f8151753 <unknown>\n#8 0x5575f811d38e <unknown>\n#9 0x5575f811eb51 <unknown>\n#10 0x5575f85dc76b <unknown>\n#11 0x5575f85e06f2 <unknown>\n#12 0x5575f85c88fc <unknown>\n#13 0x5575f85e12e4 <unknown>\n#14 0x5575f85ac9cf <unknown>\n#15 0x5575f8601cd8 <unknown>\n#16 0x5575f8601eb6 <unknown>\n#17 0x5575f8611f16 <unknown>\n#18 0x7fa15d09caa4 <unknown>\n#19 0x7fa15d129c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class PUBLICRECORDPROPERTYBUILDING is not mapped","time":{"start":1739698260128,"stop":1739698260128,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739698260164,"stop":1739698260289,"duration":125},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"8"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739698260289,"stop":1739698280342,"duration":20053},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x5575f861309a <unknown>\n#1 0x5575f80da8b0 <unknown>\n#2 0x5575f812bb99 <unknown>\n#3 0x5575f812bd21 <unknown>\n#4 0x5575f817a854 <unknown>\n#5 0x5575f81519ad <unknown>\n#6 0x5575f8177b8b <unknown>\n#7 0x5575f8151753 <unknown>\n#8 0x5575f811d38e <unknown>\n#9 0x5575f811eb51 <unknown>\n#10 0x5575f85dc76b <unknown>\n#11 0x5575f85e06f2 <unknown>\n#12 0x5575f85c88fc <unknown>\n#13 0x5575f85e12e4 <unknown>\n#14 0x5575f85ac9cf <unknown>\n#15 0x5575f8601cd8 <unknown>\n#16 0x5575f8601eb6 <unknown>\n#17 0x5575f8611f16 <unknown>\n#18 0x7fa15d09caa4 <unknown>\n#19 0x7fa15d129c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class PUBLICRECORDPROPERTYLASTMARKETSALE is not mapped","time":{"start":1739698280343,"stop":1739698280343,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739698280379,"stop":1739698280508,"duration":129},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"9"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739698280509,"stop":1739698300568,"duration":20059},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x5575f861309a <unknown>\n#1 0x5575f80da8b0 <unknown>\n#2 0x5575f812bb99 <unknown>\n#3 0x5575f812bd21 <unknown>\n#4 0x5575f817a854 <unknown>\n#5 0x5575f81519ad <unknown>\n#6 0x5575f8177b8b <unknown>\n#7 0x5575f8151753 <unknown>\n#8 0x5575f811d38e <unknown>\n#9 0x5575f811eb51 <unknown>\n#10 0x5575f85dc76b <unknown>\n#11 0x5575f85e06f2 <unknown>\n#12 0x5575f85c88fc <unknown>\n#13 0x5575f85e12e4 <unknown>\n#14 0x5575f85ac9cf <unknown>\n#15 0x5575f8601cd8 <unknown>\n#16 0x5575f8601eb6 <unknown>\n#17 0x5575f8611f16 <unknown>\n#18 0x7fa15d09caa4 <unknown>\n#19 0x7fa15d129c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class PUBLICRECORDPROPERTYLOCATION is not mapped","time":{"start":1739698300569,"stop":1739698300569,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739698300601,"stop":1739698300732,"duration":131},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"10"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739698300732,"stop":1739698320785,"duration":20053},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x5575f861309a <unknown>\n#1 0x5575f80da8b0 <unknown>\n#2 0x5575f812bb99 <unknown>\n#3 0x5575f812bd21 <unknown>\n#4 0x5575f817a854 <unknown>\n#5 0x5575f81519ad <unknown>\n#6 0x5575f8177b8b <unknown>\n#7 0x5575f8151753 <unknown>\n#8 0x5575f811d38e <unknown>\n#9 0x5575f811eb51 <unknown>\n#10 0x5575f85dc76b <unknown>\n#11 0x5575f85e06f2 <unknown>\n#12 0x5575f85c88fc <unknown>\n#13 0x5575f85e12e4 <unknown>\n#14 0x5575f85ac9cf <unknown>\n#15 0x5575f8601cd8 <unknown>\n#16 0x5575f8601eb6 <unknown>\n#17 0x5575f8611f16 <unknown>\n#18 0x7fa15d09caa4 <unknown>\n#19 0x7fa15d129c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class PUBLICRECORDPROPERTYOWNERSHIP is not mapped","time":{"start":1739698320786,"stop":1739698320786,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739698320819,"stop":1739698320951,"duration":132},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"11"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739698320951,"stop":1739698340996,"duration":20045},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x5575f861309a <unknown>\n#1 0x5575f80da8b0 <unknown>\n#2 0x5575f812bb99 <unknown>\n#3 0x5575f812bd21 <unknown>\n#4 0x5575f817a854 <unknown>\n#5 0x5575f81519ad <unknown>\n#6 0x5575f8177b8b <unknown>\n#7 0x5575f8151753 <unknown>\n#8 0x5575f811d38e <unknown>\n#9 0x5575f811eb51 <unknown>\n#10 0x5575f85dc76b <unknown>\n#11 0x5575f85e06f2 <unknown>\n#12 0x5575f85c88fc <unknown>\n#13 0x5575f85e12e4 <unknown>\n#14 0x5575f85ac9cf <unknown>\n#15 0x5575f8601cd8 <unknown>\n#16 0x5575f8601eb6 <unknown>\n#17 0x5575f8611f16 <unknown>\n#18 0x7fa15d09caa4 <unknown>\n#19 0x7fa15d129c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class PUBLICRECORDPROPERTYOWNERSHIPTRANSFER is not mapped","time":{"start":1739698340997,"stop":1739698340997,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739698341030,"stop":1739698341168,"duration":138},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"12"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739698341169,"stop":1739698361194,"duration":20025},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x5575f861309a <unknown>\n#1 0x5575f80da8b0 <unknown>\n#2 0x5575f812bb99 <unknown>\n#3 0x5575f812bd21 <unknown>\n#4 0x5575f817a854 <unknown>\n#5 0x5575f81519ad <unknown>\n#6 0x5575f8177b8b <unknown>\n#7 0x5575f8151753 <unknown>\n#8 0x5575f811d38e <unknown>\n#9 0x5575f811eb51 <unknown>\n#10 0x5575f85dc76b <unknown>\n#11 0x5575f85e06f2 <unknown>\n#12 0x5575f85c88fc <unknown>\n#13 0x5575f85e12e4 <unknown>\n#14 0x5575f85ac9cf <unknown>\n#15 0x5575f8601cd8 <unknown>\n#16 0x5575f8601eb6 <unknown>\n#17 0x5575f8611f16 <unknown>\n#18 0x7fa15d09caa4 <unknown>\n#19 0x7fa15d129c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class PUBLICRECORDPROPERTYPRIORSALE is not mapped","time":{"start":1739698361195,"stop":1739698361195,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739698361227,"stop":1739698361367,"duration":140},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"13"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739698361367,"stop":1739698381406,"duration":20039},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x5575f861309a <unknown>\n#1 0x5575f80da8b0 <unknown>\n#2 0x5575f812bb99 <unknown>\n#3 0x5575f812bd21 <unknown>\n#4 0x5575f817a854 <unknown>\n#5 0x5575f81519ad <unknown>\n#6 0x5575f8177b8b <unknown>\n#7 0x5575f8151753 <unknown>\n#8 0x5575f811d38e <unknown>\n#9 0x5575f811eb51 <unknown>\n#10 0x5575f85dc76b <unknown>\n#11 0x5575f85e06f2 <unknown>\n#12 0x5575f85c88fc <unknown>\n#13 0x5575f85e12e4 <unknown>\n#14 0x5575f85ac9cf <unknown>\n#15 0x5575f8601cd8 <unknown>\n#16 0x5575f8601eb6 <unknown>\n#17 0x5575f8611f16 <unknown>\n#18 0x7fa15d09caa4 <unknown>\n#19 0x7fa15d129c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class PUBLICRECORDPROPERTYSEARCH is not mapped","time":{"start":1739698381407,"stop":1739698381407,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739698381441,"stop":1739698381584,"duration":143},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"14"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739698381584,"stop":1739698401633,"duration":20049},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x5575f861309a <unknown>\n#1 0x5575f80da8b0 <unknown>\n#2 0x5575f812bb99 <unknown>\n#3 0x5575f812bd21 <unknown>\n#4 0x5575f817a854 <unknown>\n#5 0x5575f81519ad <unknown>\n#6 0x5575f8177b8b <unknown>\n#7 0x5575f8151753 <unknown>\n#8 0x5575f811d38e <unknown>\n#9 0x5575f811eb51 <unknown>\n#10 0x5575f85dc76b <unknown>\n#11 0x5575f85e06f2 <unknown>\n#12 0x5575f85c88fc <unknown>\n#13 0x5575f85e12e4 <unknown>\n#14 0x5575f85ac9cf <unknown>\n#15 0x5575f8601cd8 <unknown>\n#16 0x5575f8601eb6 <unknown>\n#17 0x5575f8611f16 <unknown>\n#18 0x7fa15d09caa4 <unknown>\n#19 0x7fa15d129c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class PUBLICRECORDPROPERTYSITE is not mapped","time":{"start":1739698401634,"stop":1739698401634,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Select main metadata","time":{"start":1739698401669,"stop":1739698401816,"duration":147},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"metadata","value":"15"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Implicit wait metadata","time":{"start":1739698401816,"stop":1739698421872,"duration":20056},"status":"broken","statusMessage":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"css selector\",\"selector\":\"#listing_mapper_list_category__0\"}\n  (Session info: chrome=133.0.6943.98); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x5575f861309a <unknown>\n#1 0x5575f80da8b0 <unknown>\n#2 0x5575f812bb99 <unknown>\n#3 0x5575f812bd21 <unknown>\n#4 0x5575f817a854 <unknown>\n#5 0x5575f81519ad <unknown>\n#6 0x5575f8177b8b <unknown>\n#7 0x5575f8151753 <unknown>\n#8 0x5575f811d38e <unknown>\n#9 0x5575f811eb51 <unknown>\n#10 0x5575f85dc76b <unknown>\n#11 0x5575f85e06f2 <unknown>\n#12 0x5575f85c88fc <unknown>\n#13 0x5575f85e12e4 <unknown>\n#14 0x5575f85ac9cf <unknown>\n#15 0x5575f8601cd8 <unknown>\n#16 0x5575f8601eb6 <unknown>\n#17 0x5575f8611f16 <unknown>\n#18 0x7fa15d09caa4 <unknown>\n#19 0x7fa15d129c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/slp_main.py\", line 63, in impl_wait_metadata\n    ListComponent(self.driver).get_map_filed(IMPLIS_WAIT_MAP)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/eim-map-test/eim-map-test/SLP/ui/PageObjects/SLPMain/listing_component.py\", line 46, in get_map_filed\n    return self.node.find_element(*xpath)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.2/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Looks like the class PUBLICRECORDPROPERTYTAXASSESSMENT is not mapped","time":{"start":1739698421873,"stop":1739698421873,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":48,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"_unittest_setUpClass_fixture_TestPromotionChecklist::0","time":{"start":1739706488545,"stop":1739706488545,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"@pytest.mark.filterwarnings('ignore::DeprecationWarning')"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_checklist"},{"name":"subSuite","value":"TestPromotionChecklist"},{"name":"host","value":"fv-az1779-524"},{"name":"thread","value":"3377-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_checklist"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"tags":["@pytest.mark.filterwarnings('ignore::DeprecationWarning')"]},"source":"e95299886c917a92.json","parameterValues":[]}